lib/srv/db/cloud/watchers/azure.go

```diff
@@ -268,3 +258,35 @@ func simplifyMatchers(matchers []services.AzureMatcher) []services.AzureMatcher
 	}
 	return result
 }
+
+// normalizeAzureLocation converts Azure location in differents formats to the
+// same simple format.
+//
+// This function assumes the input location is in one of the following formats:
+// - Name (the "simple" format): "northcentralusstage"
+// - Display name: "North Central US (Stage)"
+// - Regional display name: "(US) North Central US (Stage)"
+//
+// More details see:
+// https://azuretracks.com/2021/04/current-azure-region-names-reference/
+func normalizeAzureLocation(location string) string {
```

Comment:
I like the idea of mapping `Regional/Display Name` to API region name though  thinks get tricky in case of some regions for instance: 

`United Kingdom`          should be mapped to `uk`
`United Arab Emirates` should be mapped to `uae` 

This function logic can quickly get updated when a new standard region is added.

Reply:
good catch. The reason the conversion is needed is the Redis APIs actually use display names like "Canada Central".

Seems China is also different. Will look into this (seems there is an api to fetch regions, maybe use that in combination with a static mapping fallback)